### PR TITLE
increase timeout on systemd Subscribe method call

### PR DESCRIPTION
**Pull request by geissonator**
Unfortunately the previous attempt at fixing this via retries was unsuccessful. The retries returned an error saying a reply was pending.

This resulted in me going back to the OP940 workaround that was put in place for this and simply increasing the timeout. The retry seemed more efficient but better to have something that works.

This fix is based on 5cfeb07 without making it configurable as there is no need for that type of feature here.

Tested
- Basic good path testing to ensure no regressions

### CODE REVIEW

**Comment by geissonator** [comment]
Hot fix so no upstream for now but I'll at least send the simplified util upstream at some point.
Fixes 409858

File: utils.cpp [hunk 1]
```diff
@@ -28,26 +28,21 @@ void subscribeToSystemdSignals(sdbusplus::bus::bus& bus)
...
-                  e);
-            continue;
-        }
-        return;
+        // On OpenBMC based systems, systemd has had a few situations where it
+        // has been unable to respond to this call within the default d-bus
+        // timeout of 25 seconds. This is due to the large amount of work being
+        // done by systemd during OpenBMC startup. Set the timeout for this call
+        // to 60 seconds.
+        bus.call(method, (60 * 1000000L));
```
Comment (smccarney) [inline utils.cpp:39]: For the upstream commit, might be clearer to use std::chrono_literals and SdBusDuration instead.

**Review by smccarney — APPROVED** [review]

File: utils.cpp (hunk 1 above)
Comment (smccarney) [reply utils.cpp:39]: https://github.com/openbmc/phosphor-power/blob/master/phosphor-regulators/src/regsctl/utility.hpp#L31

Comment (geissonator) [reply utils.cpp:39]: Thanks, will do when I send upstream!

**Comment by geissonator** [comment]
Upstream: https://gerrit.openbmc.org/c/openbmc/phosphor-state-manager/+/61017

**Pull request merged**